src/server/api.php

```diff
@@ -34,14 +34,29 @@
 
 // passwords
 $AUTH = array(
-  'twitter'  => Secrets::$api['twitter'],
-  'ght'      => Secrets::$api['ght'],
-  'fluview'  => Secrets::$api['fluview'],
-  'cdc'      => Secrets::$api['cdc'],
-  'sensors'  => Secrets::$api['sensors'],
-  'quidel'   => Secrets::$api['quidel'],
-  'norostat' => Secrets::$api['norostat']
+  'twitter'        => Secrets::$api['twitter'],
+  'ght'            => Secrets::$api['ght'],
+  'fluview'        => Secrets::$api['fluview'],
+  'cdc'            => Secrets::$api['cdc'],
+  'sensors'        => Secrets::$api['sensors'],
+  'sensor_subsets' => Secrets::$api['sensor_subsets'],
+  'quidel'         => Secrets::$api['quidel'],
+  'norostat'       => Secrets::$api['norostat'],
 );
+$GRANULAR_SENSOR_AUTH_TOKENS = array(
+  'twtr' => array($AUTH['sensor_subsets']['twtr_sensor']),
+  'gft' => array($AUTH['sensor_subsets']['gft_sensor']),
+  'ght' => array($AUTH['sensor_subsets']['ght_sensors']),
+  'ghtj' => array($AUTH['sensor_subsets']['ght_sensors']),
+  'cdc' => array($AUTH['sensor_subsets']['cdc_sensor']),
+  'quidel' => array($AUTH['sensor_subsets']['quidel_sensor']),
+  'arch' => array($AUTH['sensor_subsets']['arch_sensor']),
+);
+$OPEN_SENSORS = array(
```

Comment:
Repeating here for better context: ARCH could probably be public if you wanted. I'm happy either way.

Reply:
Great!  I wasn't sure if this version of ARCH was incorporating non-fluview data, which is why I didn't make it open initially.  Not sure if EPIC requires more thought due to the nature of the input, though.

Reply:
IMO, EPIC is safe to share because it's an aggregate; no individual forecasts are shared. I'm happy to leave the decision to you and Roni.